lib/src/components/Drawer/AssetDrawer/index.tsx

```diff
@@ -12,15 +12,42 @@ import { CreateWuiProps, forwardRef } from '@/System'
 export type AssetDrawerProps = CreateWuiProps<'div', Ariakit.DialogOptions>
 
 export const AssetDrawerComponent = forwardRef<'div', AssetDrawerProps>(
-  ({ children, maxWidth = 820, store, ...rest }, ref) => {
+  (
+    {
+      children,
+      getPersistentElements: parentGetPersistentElements,
+      maxWidth = 820,
+      store,
+      ...rest
+    },
+    ref
+  ) => {
     const theme = useTheme()
 
+    const getPersistentElements = () =>
+      Array.from(
+        parentGetPersistentElements
+          ? parentGetPersistentElements()
+          : document.querySelectorAll('[data-wui-persistent]')
+      )
+
+    const hideOnInteractOutside =
+      Boolean(parentGetPersistentElements) ||
+      ((event: Event) => {
```

Comment:
Hi @SimonDegraeve 
I would extract the second condition in another variable for readability like:
```
const hideNotPersistentElement = ((event: Event) => {
      const target = event.target as HTMLElement
        const isTargetWithinPersistentElements = getPersistentElements().some(element =>
          element.contains(target)
        )
        return !isTargetWithinPersistentElements
}
```
(if possible)

Reply:
Done